### PR TITLE
KOGITO-3669 Node trigger should not be enabled for completed or aborted process instances

### PR DESCRIPTION
**Pull request by AjayJagan**
Many thanks for submitting your Pull Request :heart:! 

description: added a condition to hide NodeTrigger panel(hide for completed and aborted)

JIRA: https://issues.redhat.com/browse/KOGITO-3669

### CODE REVIEW

**Comment by AjayJagan** [comment]
jenkins please retest this

**Comment by AjayJagan** [comment]
jenkins retest this please

**Comment by AjayJagan** [comment]
jenkins retest this please

**Comment by AjayJagan** [comment]
jenkins retest this please

File: ui-packages/packages/management-console/src/components/Templates/ProcessDetailsPage/tests/ProcessDetailsPage.test.tsx [hunk 1]
```diff
@@ -496,4 +497,54 @@ describe('Process Details Page component tests', () => {
     window.location = location;
     expect(handleVariableUpdateSpy).toHaveBeenCalled();
   });
+  it('test node tirgger presence', async () => {
```
Comment (nmirasch) [inline ui-packages/packages/management-console/src/components/Templates/ProcessDetailsPage/tests/ProcessDetailsPage.test.tsx]: @AjayJagan typo in test name :)

**Review by nmirasch — APPROVED** [review]
just a minor comment on test name, It looks good and works fine.Thanks @AjayJagan

File: ui-packages/packages/management-console/src/components/Templates/ProcessDetailsPage/tests/ProcessDetailsPage.test.tsx (hunk 1 above)
Comment (AjayJagan) [reply ui-packages/packages/management-console/src/components/Templates/ProcessDetailsPage/tests/ProcessDetailsPage.test.tsx]: done

**Review by Sara4994 — APPROVED** [review]

**Review by jstastny-cz — APPROVED** [review]
LGTM

**Pull request merged**